website/docs/components/table/partials/specifications/anatomy.md

```diff
@@ -18,4 +18,22 @@
 |--------------|----------|
 | Cell content | Required |
 | Icon         | Optional |
+| Container    | Required |
+
+### Table header select 
+
+![Table header selection anatomy](/assets/components/table/table-header-select-anatomy.png)
+
+| Element          | Usage                                           |
+|------------------|-------------------------------------------------|
+| Checkbox         | Required                                        |
+| Container        | Required                                        |
+
+### Table cell select
+
+![Table cell selection anatomy](/assets/components/table/table-cell-select-anatomy.png)
+
+| Element      | Usage    |
+|--------------|----------|
+| Checkbox     | Required |
```

Comment:
I dropped a [comment](https://www.figma.com/file/42LK10XbP5IERhzzgMOiI2?node-id=478:37646&mode=design#654192259) in Figma about these new anatomy images.

Reply:
Thank you! I'll push those changes up

Reply:
@andgen404 if this has been addressed, can you resolve the conversation? (and ask @heatherlarsen to re-review otherwise the previous "require modifications" blocks the merging)

Reply:
@andgen404 I'm still seeing some cleanup that needs to take place on these tables, so I left a few comments around those. Let me know when those are addressed and I'll take another look.